### ISSUE TITLE
TypeError: ArchiverWebpackPlugin is not a constructor

### ISSUE THREAD

**Issue by piotrkochan** [tanem/archiver-webpack-plugin#117]
Given simple webpack script:

```
const webpack = require("webpack"),
  ArchiverWebpackPlugin = require('@tanem/archiver-webpack-plugin'),
  env = require("./env"),
  config = require("../webpack.config");

if (env.ARCHIVE_DIST) {
  config.plugins.push(
    new ArchiverWebpackPlugin('zip', {
      destpath: 'dist',
      filename: env.TARGET,
      formatOptions: {zip: true},
      globOptions: {dot: true, ignore: '*.map'},
      globPattern: '*'
    })
  );
}

webpack(
  config,
  function (err) {
    if (err) throw err;
  }
);
```

generates error:
```
utils/build.js:8
    new ArchiverWebpackPlugin('zip', {
    ^

TypeError: ArchiverWebpackPlugin is not a constructor
    at Object.<anonymous> (/.../utils/build.js:8:5)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

node:
```
node -v
v12.16.1
```

**Comment by tanem** [comment]
Hey @piotrkochan, thanks for bringing this to my attention 🙏 

This is related to how things have been built/exported. Try:

```diff
-ArchiverWebpackPlugin = require('@tanem/archiver-webpack-plugin'),
+ArchiverWebpackPlugin = require('@tanem/archiver-webpack-plugin').default,
```

I'll release a new version of the library that uses named exports shortly.

**Comment by tanem** [comment]
Just released `v4.0.0` which uses named exports. If you install that you should be able to do:

```diff
-const ArchiverWebpackPlugin = require('@tanem/archiver-webpack-plugin');
+const { ArchiverWebpackPlugin } = require('@tanem/archiver-webpack-plugin');
```

**Comment by tanem** [comment]
Closing as this should be resolved now.
